test-requirements.txt

```diff
@@ -1,2 +1,4 @@
+-c https://raw.githubusercontent.com/ska-sa/katsdpdockerbase/master/docker-base-build/base-requirements.txt
```

Comment:
Was the base requirements implicit before for test requirements?

Reply:
And in fact it still is, but this allows `pip install -r test-requirements.txt` to obtain the right versions.